### ISSUE TITLE
Out of stock not last

### ISSUE THREAD

**Issue by gino2014** [bikashkaushik/magento2-outOfStockLast#1]
Hello Bikash Kaushik,

first of all thanks for offering your work/extension here. We installed and switched on your extension at backend. Made a new reindex of all indexes (php bin/magento indexer:reindex) but the products out of stock have not been moved to last. 
We are using M 2.4.2 bit PHP 7.4.x and Elasticsearch. Maybe you have any suggestion. 
Kind regards
Gino

**Comment by purvid0204** [comment]
Hello Gino,

Have you find any solution?

**Comment by mnshankar** [comment]
True.. this does not work.. If any one has a working solution ( M 2.4.2 bit PHP 7.4.x and Elasticsearch), please share.

**Comment by web5exposure** [comment]
For me it's also not working..

Magento 2.4.3
PHP 7.4
Elasticsearch 7

**Comment by rjm87** [comment]
Also not working with Magento 2.4.3 here.

For anyone searching here, this extension does; https://github.com/tuyennn/magento2-outofstockatlast

**Comment by anasshl** [comment]
For me also is not working.
@bikashkaushik could you please provide a fix for this issue. Thank you ?

**Comment by bromios777** [comment]
Worked fine for me in Magento 2.3.5 p1

Note: remember that you need after to run magento upgrade in your ssh panel to login in magento admin >> stores>> catalog >> and to enable this extension to Yes
Thank you very much Gino2014
